docs/redirects.yaml

```diff
@@ -0,0 +1,35 @@
+"en/latest/":
```

Comment:
Once 1.10.0 is released, the 1.9.0 will have a link to the corresponding page in latest, which doesn't currently exist, but this introduces a redirect for it.